test/ui/FoldingTest.ts

```diff
@@ -104,45 +106,66 @@ export function FoldingTest() {
       });
     });
 
-    describe('expand', function () {
+    describe('expand', () => {
       var queryData: ISimulateQueryData;
 
-      beforeEach(function () {
+      beforeEach(() => {
         test = Mock.optionsComponentSetup<Folding, IFoldingOptions>(Folding, {
           field: '@fieldname',
           maximumExpandedResults: 7
         });
         queryData = Simulate.query(test.env, { query: { q: 'foo bar' }, results: fakeResults });
       });
 
-      afterEach(function () {
+      afterEach(() => {
         test = null;
         queryData = null;
       });
 
-      it('should perform query with expected expression when moreResults is called', function () {
+      it('should copy the original query, minus some property', () => {
```

Comment:
I feel like this test does too much. It is easier to understand what the test does if it's trying to only test one thing at a time. For example, having a test that only verifies that the `q` parameter is rewritten and that the `aq` is overwritten would give those tests a clearer intent. I think having a test that fails for a single reason is always ideal.